public/i18n/en.json

```diff
@@ -0,0 +1,3 @@
+{
+  "query-builder-run-query": "Run query"
+}
```

Comment:
as the documentation ( https://cli.vuejs.org/guide/html-and-static-assets.html#url-transform-rules ) suggests, using public folder directly, might not be the best approach. 
what are the arguments we have for not making it part of the module ?

Reply:
I understand but OTOH, if we want to add it to webpack, we have to add all languages to it and it would make it explode, I highly recommand keeping this part as it is (given the special way i18n works in Wikimedia).